app/views/distributions/_form.html.erb

```diff
@@ -1,37 +1,41 @@
-<div class="box-body">
-  <%= f.simple_fields_for :request do |r| %>
-    <%= r.input :id, as: :hidden %>
-  <% end %>
-  <%= f.association :partner,
-                    collection: current_organization.partners.alphabetized,
-                    label: "Partner",
-                    error: "Which partner is this distribution going to?" %>
+<%= simple_form_for distribution, html: {class: "storage-location-required"}, wrapper_mappings: { datetime: :custom_multi_select } do |f| %>
```

Comment:
The main thing done here is move the call to `simple_form_for` from the parent view to here so that we can tell turbo to reload the form and the contents of the body using just the distribution.

Plus -- the `url` doesn't have to be specified since it is already handled via simple_form_for